style.css

```diff
@@ -401,3 +402,120 @@ h4 {
 .btn:focus {
   background-color: #008552;
 }
+
+.user-form > p {
+  width: 327px;
+  height: 363px;
+  font-family: "Inter", sans-serif;
+  font-style: normal;
+  font-weight: 700;
+  font-size: 40px;
+  line-height: 52px;
+  text-align: center;
+  color: #172b4d;
+  margin-top: 54px;
+  margin-bottom: 48px;
+}
+
+.user-form ul {
+  list-style: none;
+}
+
+.user-form > ul {
+  display: flex;
+  flex-direction: column;
+  list-style: none;
+}
+
+input {
+  width: 295px;
+  height: 18px;
+  font-family: "Inter", sans-serif;
+  font-style: normal;
+  font-weight: 400;
+  font-size: 15px;
+  line-height: 24px;
+  align-items: center;
+  color: #6b778c;
+  border: none;
+}
+
+textarea {
+  width: 303px;
+  height: 90px;
+  font-family: "Inter", sans-serif;
+  font-style: normal;
+  font-weight: 400;
+  font-size: 15px;
+  line-height: 24px;
+  align-items: center;
+  color: #42526e;
+  margin: 12px;
+  border: none;
+}
+
+.input-text {
+  width: 327px;
+  height: 48px;
+  border: 1px solid #d0d9d4;
+  border-radius: 4px;
+  box-sizing: border-box;
+  margin-bottom: 24px;
+  padding: 15px 16px 15px 16px;
+}
+
+.text-area {
+  width: 327px;
+  height: 114px;
+  border: 1px solid #60c095;
+  border-radius: 4px;
+  box-sizing: border-box;
+  margin-bottom: 24px;
+}
+
+.button {
+  width: 129px;
+  height: 48px;
+  margin-bottom: 30px;
+}
+
+.email {
+  width: 381px;
+  height: 38px;
+  font-family: "Inter", sans-serif;
+  font-style: normal;
+  font-weight: 400;
+  font-size: 20px;
+  line-height: 32px;
+  text-align: center;
+  color: #42526e;
+  margin-left: 0;
+  margin-top: 0;
+}
+
+.two {
+  margin-bottom: 20.08px;
+}
+
+.nav-icons {
+  height: 131px;
+  width: 381px;
+}
+
+form {
+  background-image: url(./ImageP1/illustration-contact-form.png);
+  background-repeat: no-repeat;
+  background-position: bottom right;
+}
+
+input:invalid {
+  border: 2px dashed red;
+}
+
+input:valid {
+  border: 2px solid #36b37f;
+}
```

Comment:
- [ ] Good job implementing the input fields with placeholders. 👍 
But there's is an issue I highly recommend you to change. I found this dashed red around the placeholder text which is not part of the required figma design. I kindly suggest removing the dashed red.

_**REQUIRED FIGMA DESIGN**_

![Screenshot (113)](https://user-images.githubusercontent.com/78104544/228452684-7365e5f4-bb49-4070-9b8e-3407f22a083f.png)


_**YOUR DESIGN**_

![Screenshot (114)](https://user-images.githubusercontent.com/78104544/228453004-59c0cf65-5469-4e43-ad78-23fbcc5bc07e.png)